src/main/infrastructure/repository/iab/GVLFactory.js

```diff
@@ -3,14 +3,38 @@ import {GVL} from '@iabtcf/core'
 export class GVLFactory {
   constructor({
     baseUrl = 'https://a.dcdn.es/borostcf/v2/vendorlist',
-    language = 'es'
+    language = 'en'
   } = {}) {
     GVL.baseUrl = baseUrl
-    GVL.latestFilename = `LATEST?language=${language}`
-    GVL.versionedFilename = `[VERSION]?language=${language}`
+    this._setStaticGVLParameters({language})
+    GVL.languageFilename = 'LATEST?language=[LANG]'
   }
 
   create({version} = {}) {
     return new GVL(version)
   }
+
+  async createWith({version, language} = {}) {
+    const gvl = this.create({version})
+    await this._changeLanguage({language, gvl})
+    return gvl
+  }
+
+  resetCaches() {
+    GVL.emptyCache()
+    GVL.emptyLanguageCache()
+  }
+
+  async _changeLanguage({language, gvl}) {
+    if (language && language !== gvl.language) {
+      this._setStaticGVLParameters({language})
+      const changeLanguage = gvl.changeLanguage(language)
```

Comment:
I guess that gvl.changeLanguage(language) is async, right? so the await shouldn't be here:
`const changeLanguage = await gvl.changeLanguage(language) 
`And why do you need to store the changeLanguage variable? It could be 
`await gvl.changeLanguage(language)`

Reply:
done.